### PR TITLE
grammar mistakes

### PR DESCRIPTION
**Pull request by MrKrishnaAgarwal**
Grammar Mistakes

### CODE REVIEW

File: docs/design/generics/goals.md [hunk 1]
```diff
@@ -461,7 +461,7 @@ complicated rules to picking an implementation when there are many candidates.
 It also has a number of benefits for users:
 
 -   It removes a way packages can conflict with each other.
--   It makes the behavior of code more consistent and predictable.
+-   It makes the behaviour of code more consistent and predictable.
```
Comment (jonmeow) [inline docs/design/generics/goals.md]: American English spellings are preferred. Please see: https://developers.google.com/style/spelling

**Review by jonmeow — CHANGES_REQUESTED** [review]

**Comment by jonmeow** [comment]
For a reminder on the style guide, please see:
https://github.com/carbon-language/carbon-lang/blob/trunk/CONTRIBUTING.md#google-docs-and-markdown

**Comment by MrKrishnaAgarwal** [comment]
> For a reminder on the style guide, please see: https://github.com/carbon-language/carbon-lang/blob/trunk/CONTRIBUTING.md#google-docs-and-markdown

Sure

**Comment by MrKrishnaAgarwal** [comment]
Are other changes fine or I can close this pull request

**Comment by MrKrishnaAgarwal** [comment]
Should I do any other changes you want or is it fine?

**Comment by MrKrishnaAgarwal** [comment]
@jonmeow 

What happened, please respond 

If other things are fine

**Comment by jonmeow** [comment]
@MrKrishnaAgarwal I'm going to merge this now, but given your multiple pings over the past day, I want to clearly communicate review priorities in order to hopefully reduce your confusion in the future.

While those of us working on Carbon are fairly active, there are many contributors to Carbon and we're trying to balance all of them. Your contributions, while helpful, also need corrections. If you would take more time to make sure your proposed changes are clean of issues before sending them off, they will get in faster. When we need to correct your edits, you need to expect that we aren't waiting on you to respond, but are instead going to work on other things and come back to your change at some point later. That may not be the same day, and it may not even be the next day.

Does that make sense?

**Review by jonmeow — APPROVED** [review]

**Comment by MrKrishnaAgarwal** [comment]
Very sorry,
I'll not ping and I'll wait patiently

Sorry for the inconvenience and Thanks

**Pull request merged**